var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -205,7 +207,9 @@ class Trilinos(CMakePackage):
     depends_on('mpi')
     depends_on('netcdf+mpi', when="~pnetcdf")
     depends_on('netcdf+mpi+parallel-netcdf', when="+pnetcdf@master,12.12.1:")
+    depends_on('parallel-netcdf', when="+pnetcdf@master,12.12.1:")
```

Comment:
I don’t see where that is used,  this dependency should be there in the graph already from the previous line

Reply:
Ah I just realized I added it because I was using netcdf as an external in a certain case and then it wasn't finding parallel-netcdf in the graph. I guess the fix is probably to list an external netcdf as netcdf+parallel-netcdf.

Reply:
> I guess the fix is probably to list an external netcdf as netcdf+parallel-netcdf.

that should hopefully work.